.java-version

```diff
@@ -1 +1 @@
-17.0.9
```

Comment:
This updates the docker JVM version @mosabua

Reply:
oh .. cool